apps/prairielearn/src/middlewares/AuthzAccessMismatch.tsx

```diff
@@ -158,55 +178,35 @@ export function AuthzAccessMismatch({
   return (
     <main id="content" class="container">
       <div class="card mb-4">
-        <div class="card-header bg-danger text-white">Insufficient access</div>
+        <div class="card-header bg-danger text-white">
+          <h1>Effective user has insufficient access</h1>
+        </div>
         <div class="card-body">
-          <h2 class="mb-3 h4">Effective user has insufficient access</h2>
+          <p>{errorExplanation}</p>
           <p>
-            The
+            The current effective user{' '}
             {authzUser ? (
-              <OverlayTrigger
-                overlay={
-                  <Tooltip>
-                    {authzUser.name} ({authzUser.uid})
-                  </Tooltip>
-                }
-              >
-                <button type="button" class="btn btn-link link-secondary p-0 mx-1 align-baseline">
-                  current effective user
-                </button>
-              </OverlayTrigger>
+              <strong>
+                {authzUser.name} ({authzUser.uid})
```

Comment:
TODO: should handle users with empty names. We do that elsewhere:

https://github.com/PrairieLearn/PrairieLearn/blob/f56a37578d3c6d2dedf1442cb02fc1e0c060bcaf/apps/prairielearn/src/components/InstructorInfoPanel.tsx#L343-L345

Reply:
I addressed this in d071134.